cpp/src/arrow/compute/kernels/hash_aggregate_test.cc

```diff
@@ -1425,6 +1425,51 @@ TEST(GroupBy, MinMaxDecimal) {
   }
 }
 
+TEST(GroupBy, MinOrMax) {
+  auto table =
```

Comment:
Maybe include tests of only `hash_min` and only `hash_max`.

Reply:
I'm not sure there's a need to test the kernels separately, since they're being run separately by the nodes - I group them mostly because it's a lot of vertical visual noise to duplicate these tests otherwise (we should probably clean them up at some point).

Reply:
I agree (thus the "maybe"), if it already hitting those code paths independently which seems it is.